portfolio/src/main/webapp/script.js

```diff
@@ -150,6 +151,47 @@ function getComments()
     });
 }
 
+/** Check the login status to display the comments or force sign in */
+function checkLogin()
+{
+    var commentForm = document.getElementById('commentForm');
+    fetch('/home').then(response => response.json()).then((login) => {
+    if (login.loginStatus == 'true')
```

Comment:
In some case you use single quote (') while in other  places ("). It is better to be consistent in this.